docs/examples.rst

```diff
@@ -35,3 +35,4 @@ These examples illustrate some different possible use cases for NengoDL:
     examples/spiking-mnist
     examples/spa-retrieval
     examples/spa-memory
+    examples/lmu
```

Comment:
I think I'd put this above the SPA examples; groups nicer with spiking MNIST, and probably more likely to be looked for than the SPA examples.